asv_benchmarking/README.md

```diff
@@ -0,0 +1 @@
+Benchmarking with asv (air-speed-velocity)
```

Comment:
I think it needs some remarks on how you start it going, and how you create the webpages, etc.